NotebookAnalyzers/entities.py

```diff
@@ -11,14 +11,20 @@ class Notebook:
 
     def __init__(self, notebook_path: str):
         # Importing the notebook as a Python dictionary
-        with open(notebook_path) as f:
+        with open(config.data_path + notebook_path) as f:
```

Comment:
I believe that the input argument `notebook_path` should contain the full notebook path, rather than its name or a partial path. 

In the future, we might end up having more than one base path for target notebooks. E.g., we might choose to save repos to a different base path. In such cases, handling paths should not be a responsibility of these entity classes.